web_accounts_list.json

```
@@ -1374,6 +1374,17 @@
          "category" : "gaming",
          "valid" : true
       },
+      {
+         "name" : "Quora",
+         "check_uri" : "https://{account}.quora.com/profile/",
+         "account_existence_code" : "200",
+         "account_existence_string" : " - Quora</title>",
+         "account_missing_string" : "",

```

```suggestion
         "account_missing_string" : "<title>Quora - ",
```

The website redirects to `https://www.quora.com/profile/`